swagger_spec_validator/validator20.py

```diff
@@ -35,10 +36,14 @@ def validate_spec(spec_json, _spec_url=None):
     """
     validate_json(spec_json, 'schemas/v2.0/schema.json')
 
+    # Dereference all $refs so we don't have to deal with them
+    fix_malformed_model_refs(spec_json)
```

Comment:
Can you quantify the incurred latency penalty of L40-L42 getting called again? For something that is functional, I don't see the benefit of any further changes given the less than ideal current state of affairs is pretty well documented.